packages/grid/src/GridRenderer.ts

```diff
@@ -1060,7 +1060,8 @@ export class GridRenderer {
     context: CanvasRenderingContext2D,
     state: GridRenderState,
     column: VisibleIndex,
-    row: VisibleIndex
+    row: VisibleIndex,
+    textOverride?: string
```

Comment:
I think having a `drawCellText` method may make more sense... so something like:

```
drawCellContent(context, state, column, row) {
  const text = model.textForCell(modelColumn, modelRow);
  drawCellText(context, state, column, row, text);
}

drawCellText(context, state, column, row, text) {
   // ... all the contents of drawCellContent, except you shouldn't need the model row/column or to fetch anything from the model
}
```

Let me know if that makes sense

Reply:
I don't think this will make any difference/improvements really. There's a lot of variables used in `drawCellText` if it's split out